src/realm/array.cpp

```diff
@@ -1296,27 +1294,18 @@ MemRef Array::create(Type type, bool context_flag, WidthType width_type, size_t
 
 int_fast64_t Array::lbound_for_width(size_t width) noexcept
 {
-    if (width == 0) {
-        return 0;
-    }
-    else if (width == 1) {
-        return 0;
+    if (width == 32) {
+        return -0x80000000LL;
```

Comment:
I know it's immaterial to this PR, but why not return `std::numeric_limits<int32_t>::min()`?

Reply:
This line of code belongs to a different time.